js/core/option_manager.js

```diff
@@ -104,8 +104,8 @@ export class OptionManager {
 
     _normalizeName(name) {
         if(name) {
-            for(let deprecateName of this._deprecateNames) {
-                if(deprecateName === name) {
+            for(let i = 0; i < this._deprecateNames.length; i++) {
```

Comment:
Maybe change this logic by using something like `reduce()`?

Reply:
We break cycle using `return`. Does `reduce()` fit to us in this case?

Reply:
Maybe `find` can help us

Reply:
I did't find any use of `Array.find` in code. It seems that it may not work without polyfills (like `for...of` cycle)

Reply:
https://forum.vuejs.org/t/using-polyfill-array-find-for-ie11/40493